### ISSUE TITLE
Mutex sign with detector-like "zones"

### ISSUE THREAD

**Issue by Bevan0** [bergerhealer/TrainCarts#453]
### Description
Basically mutex zones set by detector signs. You place two of these signs like a detector and it creates a mutex zone in the bounds of that detector area.

### Examples
Fast-setup signalling systems, spacing between automatic trains with different destinations and timetables

**Comment by bergerkiller** [comment]
I agree this should be added, believe its somewhere on the discord in the suggestions area or wip. Dunno yet when I will add it though.

**Comment by bergerkiller** [comment]
This actually exists now as the [pathing mutex](https://wiki.traincarts.net/p/TrainCarts/Signs/Mutex#Pathing_Mutex)
